### PR TITLE
[de_DE] Company\JobTitle: list of common german job titles

### PR DESCRIPTION
**Pull request by amacado**

### CODE REVIEW

File: src/Faker/Provider/de_DE/Company.php [hunk 1]
```diff
@@ -11,5 +11,14 @@ class Company extends \Faker\Provider\Company
         '{{lastName}}',
     );
 
+    /**
+     * @link http://www.personalseite.de/information/titel.htm
+     */
+    protected static $jobTitleFormat = array(
+        'Abteilungsdirektor', 'Arbeitsdirektor', 'Aufsichtsrat', 'Beirat', 'Bereichsleiter', 'Betriebsleiter', 'Finanzvorstand', 'Geschäftsführender Gesellschafter', 'Geschäftsführer', 'Gesellschafter',
```
Comment (localheinz) [inline src/Faker/Provider/de_DE/Company.php]: Apart from possible copyright issues, can you trim the job titles, please? There are a few entries with trailing whitespace.

Comment (amacado) [reply src/Faker/Provider/de_DE/Company.php]: Hi, thanks for your Feedback, but the Job titles including a whitespace does not make sense without the whitespace. Therefore need to be removed completly, but what is the reason that whitespace is not allowed?

File: src/Faker/Provider/de_DE/Company.php [hunk 2]
```diff
@@ -11,5 +11,14 @@ class Company extends \Faker\Provider\Company
...
+        'Handlungsbevollmächtigter', 'Kaufmännischer Vorstand', 'Leiter Rechtsabteilung', 'Mitglied des Aufsichtsrats', 'Personalleiter', 'Prokurist', 'Sellvertretender Vorsitzender des Vorstandes',
+        'Vorsitzender der Geschäftsführung', 'Vorsitzender des Aufsichtsrats', 'Vorsitzender des Vorstandes ', 'Vorstand Personal ', 'Vorstand Technik ', 'Vorstand Vertrieb ', 'Vorstandsmitglied ', 'Werksleiter'
```
Comment (localheinz) [inline src/Faker/Provider/de_DE/Company.php]:
```suggestion
        'Vorsitzender der Geschäftsführung', 'Vorsitzender des Aufsichtsrats', 'Vorsitzender des Vorstandes ', 'Vorstand Personal ', 'Vorstand Technik', 'Vorstand Vertrieb', 'Vorstandsmitglied', 'Werksleiter'
```

Comment (amacado) [reply src/Faker/Provider/de_DE/Company.php]: Now I see what you mean ;-) Sorry for the misunderstanding and thank you for the hint

**Comment by localheinz** [comment]
Thank you, @amacado and @pimjansen!

**Pull request merged**